acra/acra-in-depth/data-structures/acrastruct-validator.md

```diff
@@ -0,0 +1,53 @@
+---
+title: AcraStruct validator
+bookCollapseSection: true
+---
+
+## AcraStruct validator
+
+To decrypt AcraStructs generated by AcraWriter in your application, you can also use our [AcraStruct validator](/simulator/acra/). It works as AcraServer / AcraTranslator and uses private decryption keys.
```

Comment:
I believe that either *all* relative URL paths should use `{{< ref "/my/relative/path" >}}` syntax, or none of them. Including URLs of images.

Reply:
Oh! Now I understand this magic. Thanks 👍